stdlib/map.mli

```diff
@@ -179,6 +179,25 @@ module type S =
        @before 4.03 Physical equality was not ensured.
      *)
 
+    val filter_map: (key -> 'a -> 'b option) -> 'a t -> 'b t
+    (** [filter_map f m] applies the function [f] to every element of
+        [m], and builds a map from the results: for each binding
+        [(k, v)] in the input map, if [f v] is [None] then [k] is
+        not in the result, if [f v] is [Some v'] then the binding
+        [(k, v')] is in the output map.
```

Comment:
`[f v]` should be `[f k v]` (twice). Similarly, maybe `applies the function [f] to every binding of [m]` instead of `every element`?

Reply:
Done, thanks! (Plus I used an explicit list for the two cases, which reads better than a paragraph of text.)